packages/x-components/src/x-modules/facets/components/__tests__/preselected-filters.spec.ts

```diff
@@ -24,12 +25,18 @@ function renderPreselectedFilters({
     },
     localVue,
     mocks: {
-      $x: {
+      emit: {
         emit
       }
     }
   });
 
+  const eventMetadata = {
+    moduleName: null,
+    location: 'none',
+    replaceable: true
+  };
+
```

Comment:
If you don't care what the metadata is in the tests, you can use `expect.any(Object)`.
Remove this and use that instead.